src/compiler/operations.md

```diff
@@ -1,6 +1,8 @@
 # Operations
 
-Here is a list of recurring tasks to support the compiler team and help keep things moving forward. Ideally run through this list every week. If there are blockers or doubts, after having acquired the right context, don't hesitate to ping people around. Keep in mind that contributors are the best resource of the project and we want to be mindful of their time.
+"Operations" is a part of the Compiler Team that takes care of organizational work and in general help things moving forward. T-compiler ops lives on Zulip under [#t-compiler/ops](#).
```

Comment:
Suggestion:

```suggestion
"Operations" is a part of the Compiler Team that takes care of organizational and maintenance work and in general help things moving forward. T-compiler ops lives on Zulip under [#t-compiler/ops](#).
```

"organizational work" is IMO underselling the value of such work, but anyway.

Reply:
ah here it comes again the imposter syndrome :sweat_smile: